### PR TITLE
Mimir 385 frontpage statistics list

### PR DESCRIPTION
**Pull request by itemodahl**

### CODE REVIEW

File: src/main/resources/assets/styles/_nextStatisticsReleases.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  h3 {
+    font-size: 28px;
+    margin-bottom: 30px;
+  }
+
+  article {
+    display: flex;
+
+    &.first .releaseList {
+        border-top: 2px solid black;
```
Comment (Garlov) [inline src/main/resources/assets/styles/_nextStatisticsReleases.scss:13]: see jira for list of style missmatches

**Review by Garlov — CHANGES_REQUESTED** [review]

**Review by Garlov — APPROVED** [review]
👍

File: src/main/resources/site/parts/releasedStatistics/releasedStatistics.ts [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+    })
+    groupedByYearMonthAndDay[year] = groupedByMonthAndDay
+  })
+
+  // render component
+  const reactComponent: React4xpObject = new React4xp('ReleasedStatistics')
+    .setProps({
+      releases: groupedByYearMonthAndDay
+    })
+    .setId('nextStatisticsReleases')
```
Comment (ssb-cgn) [inline src/main/resources/site/parts/releasedStatistics/releasedStatistics.ts:90]: Maybe change from nextStatisticsReleases to previousStatisticsReleases?

File: src/main/resources/site/parts/releasedStatistics/releasedStatistics.ts [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+  // All statistics published today, and fill up with previous releases.
+  const releasesFiltered: Array<StatisticInListing> = filterOnPreviousReleases(releases, numberOfReleases)
+
+  // Choose the right variant and prepare the date in a way it works with the groupBy function
+  const releasesPrepped: Array<PreparedStatistics> = releasesFiltered.map((release: StatisticInListing) => prepareRelease(release))
+
+  // group by year, then month, then day
+  const groupedByYear: GroupedBy<PreparedStatistics> = groupStatisticsByYear(releasesPrepped)
+  const groupedByMonthAndDay: GroupedBy<GroupedBy<PreparedStatistics>> = {}
+  const groupedByYearMonthAndDay: GroupedBy<GroupedBy<GroupedBy<PreparedStatistics>>> = {}
```
Comment (ssb-cgn) [inline src/main/resources/site/parts/releasedStatistics/releasedStatistics.ts:76]: This get doble dates if releases gets over 2 year

**Review by ssb-cgn — CHANGES_REQUESTED** [review]
Look at doble dates if dates from two years

File: src/main/resources/react4xp/_entries/ReleasedStatistics.jsx [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+            )
+          })}
+        </ol>
+      </article>
+    )
+  }
+
+  render() {
+    return (<section className='nextStatisticsReleases'>
+      <Title size={3}>Ny statistikk</Title>
```
Comment (ssb-cgn) [inline src/main/resources/react4xp/_entries/ReleasedStatistics.jsx:36]: Bør være en frase

File: src/main/resources/site/i18n/phrases.properties [hunk 5]
```diff
@@ -83,6 +83,14 @@ interval.M.12 = Desember
 interval.T = Termin
 interval.U = Uke
 
+period.generic = tall for {0}
```
Comment (ssb-cgn) [inline src/main/resources/site/i18n/phrases.properties:86]: Må legge til disse i alle frasefiler

File: src/main/resources/react4xp/_entries/ReleasedStatistics.jsx [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+      <article className={indexes.dayIndex === 0 && 'first'} key={`${indexes.yearIndex}-${indexes.monthIndex}-${indexes.dayIndex}`}>
+        <time dateTime={`${currentDate.year}-${currentDate.month}-${currentDate.day}`}>
+          <span className='day'>{currentDate.day}</span>
+          <span className='month'>{months[currentDate.month]}</span>
+        </time>
+        <ol className='releaseList'>
+          {currentDayReversed.map((release, index) => {
+            return (
+              <li key={index}>
+                <Link href={`/${release.shortName}`} linkType='header'>{release.name}</Link>
```
Comment (ssb-cgn) [inline src/main/resources/react4xp/_entries/ReleasedStatistics.jsx:24]: På engelsk forside må linken gå til /en/shortname

**Review by ssb-cgn — CHANGES_REQUESTED** [review]
Need english phrases and som small fixes as i comment

File: src/main/resources/site/i18n/phrases_en.properties [hunk 7]
```diff
@@ -160,6 +167,7 @@ qualityOfSeasonalAdjustment = Quality of seasonal adjustment
 specialCases = Special cases
 postingProcedures = Posting procedures
 notRelevant = Not relevant
+and = and
```
Comment (ssb-cgn) [inline src/main/resources/site/i18n/phrases_en.properties:170]: Denne mangler i 2 frasefiler

**Review by ssb-cgn — APPROVED** [review]
God job 🥇

**Pull request merged**